lib/Locale/Po4a/AsciiDoc.pm

```diff
@@ -790,7 +790,7 @@ sub parse {
             }
             @comments = ();
         } elsif ( not defined $self->{verbatim}
-            and ( $line =~ m/^([\w\d][\w\d-]*)(::)(\S*)\[(.*)\]$/ ) )
+            and ( $line =~ m/^([\w\d][\w\d-]*)(::)(\S*|\S*\{.*\}\S*)\[(.*)\]$/ ) )
```

Comment:
This only matches one transclusion. Is there any chance that several Hugo shortcodes may be used?

Reply:
In all macro examples I found, there is always one variable.
I think it would be very unusual having more (but it's possible).